tests/integration/cmd_app_test.go

```diff
@@ -82,11 +82,11 @@ var _ = Describe("odo app command tests", func() {
 			} else {
 				sourcePath = "file://./"
 			}
-			desiredCompListJSON := fmt.Sprintf(`{"kind":"List","apiVersion":"odo.openshift.io/v1alpha1","metadata":{},"items":[{"kind":"Component","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"nodejs","creationTimestamp":null, "namespace":"%s"},"spec":{"type":"nodejs","app":"app","source":"%s","env":[{"name":"DEBUG_PORT","value":"5858"}]},"status":{"state":"Pushed"}}]}`, project, sourcePath)
+			desiredCompListJSON := fmt.Sprintf(`{"kind":"ApplicationList","apiVersion":"odo.openshift.io/v1alpha1","metadata":{},"items":[{"kind":"Component","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"nodejs","creationTimestamp":null, "namespace":"%s"},"spec":{"type":"nodejs","app":"app","source":"%s","env":[{"name":"DEBUG_PORT","value":"5858"}]},"status":{"state":"Pushed"}}]}`, project, sourcePath)
```

Comment:
This should be `list` as the above command triggered is `odo list`
https://travis-ci.com/openshift/odo/jobs/239155830#L1206

Reply:
Thanks! Updated.